cmd/cstor-pool-mgmt/volumereplica/volumereplica.go

```diff
@@ -76,6 +76,7 @@ func createVolumeBuilder(cStorVolumeReplica *apis.CStorVolumeReplica, fullVolNam
 	openebsTargetIP := "io.openebs:targetip=" + cStorVolumeReplica.Spec.TargetIP
 
 	createVolAttr = append(createVolAttr, "create",
+		"-b", "4K", "-s", "-o", "compression=on",
```

Comment:
Is it fine to pass the "-o compression=on" in the middle instead of at the end ?

Reply:
this is fine